code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -129,4 +129,4 @@
 	name = "salicyclic acid pill"

```

Comment:
Can you please make it ```Salicylic acid pill (24u)``` so the name matches that of chemist created pills?

PS: The name is spelled wrong. Fixed that above.